relentless/potential/pair.py

```diff
@@ -6,10 +6,44 @@
 from .potential import PairPotential
 
 class LennardJones(PairPotential):
+    """Models particle pair interactions using the Lennard-Jones potential.
+
+    Two parameters are used: epsilon and sigma.
```

Comment:
We should expand this a bit more to include the equation and some more details. Maybe something like:

```py
.. math::

    u(r) = 4 \varepsilon\left[\left(\frac{\sigma}{r}\right)^12 - \left(\frac{\sigma}{r}\right)^12 \right]

The required coefficients per pair are:

- :math:`\varepsilon`: interaction energy
- :math:`\sigma`: interaction length scale (e.g., particle diameter)

The optional coefficients per pair are:

- ``rmin``: minimum radius, energy and force are 0 for ``r < rmin``. Ignored if ``False`` (default).
- ``rmax``: maximum radius, energy and force are 0 for ``r > rmax`` Ignored if ``False`` (default).
- ``shift``: If ``True``, shift potential to zero at ``rmax`` (default is ``False``).
```